### DISCUSSION TITLE
Checkbox to toggle visibility of slices

### DISCUSSION THREAD

**Discussion by fdinhSG** [pyvista/pyvista#5384] [Q&A]
Take the following script from the examples:

```python
import pyvista as pv
pl = pv.Plotter()
slices = pl.add_mesh(pv.Sphere())
def toggle_vis(flag):
    slices.SetVisibility(flag)
pl.add_checkbox_button_widget(toggle_vis, value=True)
pl.show()
```

This works as intended. However this stops working once I'm doing it with slices:

```python
import pyvista as pv
pl = pv.Plotter()
slices = pl.add_mesh(pv.Sphere().slice_orthogonal(x=0, y=0, z=0))
def toggle_vis(flag):
    slices.SetVisibility(flag)
pl.add_checkbox_button_widget(toggle_vis, value=True)
pl.show()
```

And the following error comes out:

    AttributeError: 'tuple' object has no attribute 'SetVisibility'

**Comment by banesullivan** [comment]
`slices` is more than one actor since `.slice_orthogonal()` is producing multiple meshes in a `MultiBlock` dataset. Try iterating over the slices. Or adding them to the scene individually, each with their own widget and callback.
